modules/installation-registry-osp-creating-custom-pvc.adoc

```diff
@@ -79,6 +79,12 @@ $ oc apply -f <pvc_file_name>
 ----
 persistentvolumeclaim/csi-pvc-imageregistry created
 ----
++
+[NOTE]
+====
+RWO accèss modes needs rolloutStrategy  set as recreate and replica count must be set as 1.
```

Comment:
I'd change this to: "RWO access mode requires `rolloutStrategy` set to `recreate` and `replica count`set to `1`.